pkg/controller/command/didexchange/command.go

```diff
@@ -118,25 +136,24 @@ type Command struct {
 // GetHandlers returns list of all commands supported by this controller command
 func (c *Command) GetHandlers() []command.Handler {
 	return []command.Handler{
-		cmdutil.NewCommandHandler(commandName, "CreateInvitation", c.CreateInvitation),
-		cmdutil.NewCommandHandler(commandName, "ReceiveInvitation", c.ReceiveInvitation),
-		cmdutil.NewCommandHandler(commandName, "AcceptInvitation", c.AcceptInvitation),
-		cmdutil.NewCommandHandler(commandName, "RemoveConnection", c.RemoveConnection),
-		cmdutil.NewCommandHandler(commandName, "QueryConnectionByID", c.QueryConnectionByID),
-		cmdutil.NewCommandHandler(commandName, "QueryConnections", c.QueryConnections),
-		cmdutil.NewCommandHandler(commandName, "AcceptExchangeRequest", c.AcceptExchangeRequest),
-		cmdutil.NewCommandHandler(commandName, "CreateImplicitInvitation", c.CreateImplicitInvitation),
+		cmdutil.NewCommandHandler(commandName, createInvitationCommandMethod, c.CreateInvitation),
+		cmdutil.NewCommandHandler(commandName, receiveInvitationCommandMethod, c.ReceiveInvitation),
+		cmdutil.NewCommandHandler(commandName, acceptInvitationCommandMethod, c.AcceptInvitation),
+		cmdutil.NewCommandHandler(commandName, removeConnectionCommandMethod, c.RemoveConnection),
+		cmdutil.NewCommandHandler(commandName, queryConnectionByIDCommandMethod, c.QueryConnectionByID),
+		cmdutil.NewCommandHandler(commandName, queryConnectionsCommandMethod, c.QueryConnections),
+		cmdutil.NewCommandHandler(commandName, acceptExchangeRequestCommandMethod, c.AcceptExchangeRequest),
+		cmdutil.NewCommandHandler(commandName, createImplicitInvitationCommandMethod, c.CreateImplicitInvitation),
 	}
 }
 
 // CreateInvitation Creates a new connection invitation.
 func (c *Command) CreateInvitation(rw io.Writer, req io.Reader) command.Error {
-	logger.Debugf("Creating connection invitation ")
-
 	var request CreateInvitationArgs
 
 	err := json.NewDecoder(req).Decode(&request)
 	if err != nil {
+		logutil.LogError(logger, commandName, createInvitationCommandMethod, err.Error())
```

Comment:
We should consider a lower log level for errors caused by the remote party.

Seems more like Info (or at most Warning for some cases).

Reply:
[Updated](https://github.com/hyperledger/aries-framework-go/compare/67880d1d3421b9f64b00f509e475f1252ec46464..6fc292b0547264d82aeddcf173e9ce562b2e6b7c).